rhoas-client/src/main/java/com/redhat/service/bridge/rhoas/exceptions/RhoasClientException.java

```diff
@@ -0,0 +1,13 @@
+package com.redhat.service.bridge.rhoas.exceptions;
+
+import java.util.List;
+
+import io.smallrye.mutiny.CompositeException;
+
+public class RhoasClientException extends RuntimeException {
```

Comment:
@kostola Any reason why these shouldn't extend our own `EventBridgeManagerException` hierarchy?

Reply:
@robpblake I can't find the exception you mentioned. It seems like it was removed in some refactoring. Am I wrong?

Reply:
@kostola It's now called `InternalPlatformException` after refactor in #183

Reply:
Done :+1: